### ISSUE TITLE
Display issues (CSS?) On Bug/Volunteer Opportunity Tracker

### ISSUE THREAD

**Issue by SarahRK** [openhatch/oh-mainline#1553]
Some issues display fine, but others have overlapping words. 

![screen shot 2015-03-08 at 2 15 31 pm](https://cloud.githubusercontent.com/assets/4283226/6547203/79aebe82-c59f-11e4-8830-dc3f2e55fcd1.png)

These occur frequently, but not exclusively, here: 
https://openhatch.org/search/?q=&project=Python&language=Python

**Comment by chums923** [comment]
I'm getting the same error with Chrome 40.0. What browser and version are you using?

**Comment by chums923** [comment]
In fact, this looks like it's happening on any page where the language being queried is longer than a couple characters. Does anyone know what the purpose of having (and in Python) even is? I mean, obviously we're on the Python page, why even display that?

**Comment by SarahRK** [comment]
No idea.

But not all Python listings contain that phrase, e.g.:
https://openhatch.org/search/?q=&project=Pitivi&language=Python

The issue isn't always present in listings that say "(and in Python)". e.g. it works here:
https://openhatch.org/search/?q=&project=GNU+Mediagoblin&language=Python
(because there is an extra line)

But not here:
https://openhatch.org/search/?q=&project=Django&language=Python

However, even in the case of GNU Mediagoblin, I suspect that we would be best off wrapping it.

**Comment by chums923** [comment]
Right, so looks like the difference in those cases is that in your second link, the 
`span class="project__name"`has a large enough height to push down the `div class="helpers"` down below the `div class="opp_data"`. That's not happening in the third link, I guess because the string just isn't long enough. 

We should probably do something where we force the `helpers` div to wrap around so it doesn't get too wide. Thoughts?

**Comment by chums923** [comment]
Anyone else have any input? I think I'm not totally understanding why we have the (and in Python) thing on the end. If that's super important we can wrap the text and have in just make a new line. If it's not, we can either hide the overflow or we can just get rid of it altogether. 

I'd like to make a PR pretty soon with a bunch of these little CSS changes so I'd appreciate any input

**Comment by SarahRK** [comment]
Sorry for not responding sooner. 

I don't know any CSS, so I'm not very useful for reviewing, but your solution sounds fine to me.

Regarding the (and in Python) thing, have you pinged the OpenHatch IRC channel? Someone there might know. If you want, I can do it.

Thank you for being awesome. Sorry I can't be of more help.

**Comment by ehashman** [comment]
Hi there @SarahRK, I took a look and I believe this is desired behaviour. When you expand bugs, the text wraps around properly:

![expanded](https://cloud.githubusercontent.com/assets/4706131/7103109/b29aa4fa-e065-11e4-828c-f0aa1cbdad55.png)

However, when bugs are collapsed, I believe they are all intended to be the same height so a word wrap wouldn't really work. May I ask what behaviour you expected/what you'd like to see this changed to? For instance, we could change text that is too long to end in '...' or something.

**Comment by paulproteus** [comment]
I just merged a pull request that relates to this, but I share @ehashman 's feeling that it's nice to have the bug objects be collapsed by default, and require a click to expand -- that's the concept behind the current design.

To my mind, the overlapping words problem should be fixed by, when the "and in Python" text would overflow, it would become, "and in Py...".

To answer the question of why we even say "and in Python", the idea is that we wanted to show "X people can mentor you in this project specifically, and Y people have indicated they're willing to mentor any project that uses the Python programming language". When Python is the language as well as the project, then this gets extremely confusing.

Having said that, if we think we should change this design, I'm A-OK with that. I just wanted to remark on what this is about.

**Comment by SarahRK** [comment]
Hi Asheesh,

Are you saying we should occasionally see things like:

Mentors: 254 (and 314 in Python)?

Because I was only seeing "(and in Python)", with no number filled in.
Either way, its current design is confusing.
